### PR TITLE
Issues/#24 provider db format variations

### PR DESCRIPTION
**Pull request by gedl**

### CODE REVIEW

File: src/main/java/technology/dice/dicewhere/provider/dbip/parsing/DbIpIpToCityCSVLineParser.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  public ParsedLine parse(RawLine line, boolean retainOriginalLine) throws LineParsingException {
+    try {
+      Iterable<String> fieldsIterable = splitter.split(line.getLine());
+      Iterator<String> fieldsIterator = fieldsIterable.iterator();
+      String rangeStartString = StringUtils.removeQuotes(fieldsIterator.next());
+      String rangeEndString = StringUtils.removeQuotes(fieldsIterator.next());
+      fieldsIterator.next(); // skip continent
+      String countryCode = fieldsIterator.next();
+      String leastSpecificDivision = fieldsIterator.hasNext() ? fieldsIterator.next() : null;
+      String city = fieldsIterator.hasNext() ? fieldsIterator.next() : null;
```
Comment (AlexKryv) [inline src/main/java/technology/dice/dicewhere/provider/dbip/parsing/DbIpIpToCityCSVLineParser.java]: null or empty string?

Comment (gedl) [reply src/main/java/technology/dice/dicewhere/provider/dbip/parsing/DbIpIpToCityCSVLineParser.java]: I am going with your previous advise of using null.
Have you changed your mind?

File: src/test/java/technology/dice/dicewhere/parsing/provider/dbip/DbIpIpToCityCSVLineParserTest.java [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  @Test
+  public void ipV6LineWithWithoutOriginalLine() throws LineParsingException {
+	  DbIpIpToCityCSVLineParser dbIpIpToCityCSVLineParser =
+			  new DbIpIpToCityCSVLineParser();
+	  String line =
+			  "2c0f:ff80::,2c0f:ff80:ffff:ffff:ffff:ffff:ffff:ffff,AF,ZA,Gauteng,\"Sandton (Woodmead)\",-26.0561,28.0696";
+	  RawLine rawLine = new RawLine(line, 1);
+	  ParsedLine parsed = dbIpIpToCityCSVLineParser.parse(new RawLine(line, 1), false);
+	  ParsedLine expected =
+			  new ParsedLine(
```
Comment (julianhowarth) [inline src/test/java/technology/dice/dicewhere/parsing/provider/dbip/DbIpIpToCityCSVLineParserTest.java]: Formatting is not correct here

File: src/main/java/technology/dice/dicewhere/provider/dbip/parsing/DbIpIpToCityCSVLineParser.java (hunk 1 above)
Comment (julianhowarth) [reply src/main/java/technology/dice/dicewhere/provider/dbip/parsing/DbIpIpToCityCSVLineParser.java]: Will be much better once it's using the [builder](#8) instead

File: src/main/java/technology/dice/dicewhere/parsing/ParsedLine.java [hunk 3]
```diff
@@ -50,4 +51,14 @@ public int hashCode() {
 
     return Objects.hash(rawLine, startIp, endIp, info);
   }
+
+	@Override
+	public String toString() {
+		return "ParsedLine{" +
```
Comment (julianhowarth) [inline src/main/java/technology/dice/dicewhere/parsing/ParsedLine.java]: format is wrong here too

**Comment by gedl** [comment]
addresses #24

**Pull request merged**